docs/en/observability/troubleshoot-uptime.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[troubleshoot-uptime]]
+= Uptime troubleshooting
+
+== Mapping issues
+
+It is possible for Heartbeat to create an index without the appropriate mappings.
```

Comment:
Could we phrase this differently? This makes it seem like a heartbeat bug. Maybe:

"There are situations in which Heartbeat data can be indexed without the correct mappings applied. These situations cannot occur with the elastic-agent configured via fleet, only with standalone heartbeat. This can occur when the underlying `heartbeat-VERSION` ILM alias is deleted manually or when heartbeat writes data through an intermediary such as Logstash without the `setup` command being run."

Reply:
We may also want to recommend agent / fleet as an alternative here

Reply:
Though they still need to delete the bad indices in that case

Reply:
> Could we phrase this differently? This makes it seem like a heartbeat bug.

Added in https://github.com/elastic/observability-docs/pull/1046/commits/5e439ac02744484eff67ea9612e817cb09c02f71.

Need to research where in Fleet docs it makes the most sense for us to link.